source/DayPicker.js

```diff
@@ -0,0 +1,153 @@
+(function (enyo, scope) {
+	/**
+	* {@link moon.DayPicker}, which extends {@link moon.ExpandablePicker}, is
+	* a drop-down picker menu that solicits day of the week from the user. 
+	*
+	* ```
+	* {kind: 'moon.DayPicker'}
+	* ```
+	*
+	* When the picker is minimized, the currently selected day are
+	* displayed as subtext below the picker label. And if the picker select every weekday,
+	* subtext will be changed 'Every Weekday' automatically.
+	*
+	* The content of representative value can be changed.
+	* ```
+	* {kind: 'moon.DayPicker', everyWeekday:'Weekdays', everyWeekend:'Weekends', everyday:'Daily'}
```

Comment:
In the example, I think we will want to encourage developers (particularly app developers) to wrap their overridden strings with `moon.$L()` so that these strings are identified and properly localized.